horace_core/sqw/@DnDBase/change_crystal.m

```diff
@@ -40,12 +40,17 @@
         rlu_corr = this_alignment.get_corr_mat(obj.proj);
         rlu_to_u = wout(i).proj.bmatrix();
         proj = wout(i).proj;
-        proj.img_offset(1:3)=rlu_corr*wout(i).img_offset(1:3)';
-        proj = proj.set_ub_inv_compat(rlu_corr/rlu_to_u);
-        wout(i).proj = proj;
+        %proj.img_offset(1:3)=rlu_corr*wout(i).img_offset(1:3)';
```

Comment:
What's the differece between l. 48 and l. 43 besides an intermediate variable? If so, do we need this comment here?

Reply:
43 is the same as 5 other rows but using `img_offset`. It is removed so can not be used, but having it would be benefitial and more clear code. As single row is summary of 5 others with clear meaning, it is worth keeping it (and adding more comments)